test/helpers/api-unit.helper.js

```diff
@@ -0,0 +1,46 @@
+// @TODO: remove when lodash can be upgraded
+import defaults from 'lodash.defaultsdeep';
+import { model as User } from '../../website/src/models/user'
+import { model as Group } from '../../website/src/models/group'
+import i18n from '../../common/script/src/i18n';
+require('coffee-script');
+i18n.translations = require('../../website/src/libs/i18n.js').translations;
+
+afterEach(() => {
+  sandbox.restore();
+});
+
+export function generateUser(options={}) {
+  return new User(options)._doc;
```

Comment:
Not sure how I feel about reaching into the private property `_doc` to generate a fake user. Seems perilous. I definitely wouldn't do this in production code.

For tests, if something about the implementation changed in the future, this would break all the tests, but this should be the only place it'd need to be adjusted to fix it.